app/models/keyword_stats.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class KeywordStats < ApplicationRecord
```

Comment:
do we still need this file @Goose97 🤔

Reply:
Removed in ce0cfdcdc4cc60406a727391eeaffee341e80b62